src/provisioner/provisioner.py

```diff
@@ -32,6 +33,14 @@ class PROVISIONING_STAGES(Enum):
     DEPLOYMENT = "deployment"
     UPGRADE = "upgrade"
 
+class MINIPROVISIONING_STAGES(Enum):
+    POST_INSTALL= "post_install"
+    PREAPRE= "prepare"
+    CONFIG= "config"
+    INIT= "init"
+
+class UPGRADEPROVISIONING_STAGES(Enum):
```

Comment:
Stages is confusing. Can we have this as 

**class UpgradeInterfaces(Enum):**

Reply:
done